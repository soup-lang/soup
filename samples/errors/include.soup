include "samples/include/error.soup" as whatever
console.log(whatever.a)
