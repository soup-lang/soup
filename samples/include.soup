include "samples/include/left.soup" as left
console.log(left.foo())
console.log(left.bar())
