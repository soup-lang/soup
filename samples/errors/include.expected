samples/include/error.soup:1
    asynchronous expression not allowed in global scope of an included file
